test/services/ArrivalMovementServiceSpec.scala

```diff
@@ -129,7 +129,7 @@ class ArrivalMovementServiceSpec extends SpecBase with IntegrationPatience {
       val messageCorrelationId = 1
       val expectedMessage      = MovementMessageWithoutState(LocalDateTime.of(dateOfPrep, timeOfPrep), MessageType.GoodsReleased, movement, messageCorrelationId)
 
-      service.makeGoodsReleasedMessage(messageCorrelationId)(movement).value mustEqual expectedMessage
+      service.makeMessage(messageCorrelationId, MessageType.GoodsReleased)(movement).value mustEqual expectedMessage
```

Comment:
I know the method is _makeMessage_ but do we need some tests for UnloadingPermission (similar to the what's in the _makeGoodsReleasedMessage_ block